src/checkpoint01/styles.css

```diff
@@ -0,0 +1,470 @@
+* {
+  background-color: #1F1F1F;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.nav {
+  position: fixed;
+  width: 100%;
+  background-color: #1F1F1F;
+  height: 60px;
+  padding-left: 8%;
+  color: #C0C0C0;
+}
+
+.gridcontainer {
+  height: 240%;
```

Comment:
change this to `height: auto;` and it'll take up the height of the contents inside the grid.